udata/harvest/backends/dcat.py

```diff
@@ -120,3 +123,62 @@ def process(self, item):
         dataset = self.get_dataset(item.remote_id)
         dataset = dataset_from_rdf(graph, dataset, node=node)
         return dataset
+
+
+class CswDcatBackend(DcatBackend):
+    display_name = 'CSW-DCAT'
+
+    def parse_graph(self, url, fmt):
+        body = '''<csw:GetRecords xmlns:csw="http://www.opengis.net/cat/csw/2.0.2"
```

Comment:
We could also consider using a lib to query a CSW endpoint, ex https://pypi.org/project/OWSLib.
Maybe we want to keep the dcat content raw instead?

Reply:
Seems like in this example and others I've tried, `csw.records` is always empty with `outputschema='http://www.w3.org/ns/dcat#'` sadly

Reply:
I've tested to run the script and not empty so not sure if there is a misunderstanding or it's related to the owslib version or something else.

Reply:
Wrong check on my side. It only works with default outputschema

Reply:
Since owslib doesn't seem to give us a satisfying result, let's keep with a raw query for now